### PR TITLE
Reward Points Mobile

### PR DESCRIPTION
**Pull request by DragonUncaged**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes:  #756

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] I have made this from my own
- [ ] I have taken help from some online resourses 
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
preview

image

### CODE REVIEW

**Comment by DEEPIKA-SUK** [comment]
@DragonUncaged make it of full width.
Screenshot (129)

**Comment by DEEPIKA-SUK** [comment]
https://deploy-preview-775--cool-front-end-templates.netlify.app/reward%20point/

**Comment by DragonUncaged** [comment]
> https://deploy-preview-775--cool-front-end-templates.netlify.app/reward%20point/

I have designed for the mobile view.
I have written Mobile it will not look good 
It will just get enlarged.
If you say i will do that..

**Comment by DEEPIKA-SUK** [comment]
> > https://deploy-preview-775--cool-front-end-templates.netlify.app/reward%20point/
> 
> I have designed for the mobile view. I have written Mobile it will not look good It will just get enlarged. If you say i will do that..

ok leave.

**Review by DEEPIKA-SUK — APPROVED** [review]

**Comment by DEEPIKA-SUK** [comment]
@DragonUncaged update the style_links.json with correct link.

File: style_links.json [hunk 1]
```diff
@@ -409,6 +409,12 @@
     {
       "name": "Chrome Extension UI",
       "link": "Chrome%20Extension%20Project/index.html"
+    },
+    {
+      
+      "name": "Reward Point",
+      "link": "https://deploy-preview-775--cool-front-end-templates.netlify.app/reward%20point/"
+
```
Comment (DragonUncaged) [inline style_links.json:417]: Don!!

**Review by DragonUncaged — COMMENTED** [review]
Done

**Review by DEEPIKA-SUK — APPROVED** [review]

**Pull request merged**